cmd/nodeagent/nodeagentmanager.go

```diff
@@ -220,11 +220,15 @@ func (n *NodeAgentManager) gameServerCreatedOrUpdated(obj *unstructured.Unstruct
 		// or that the NodeAgent crashed and we're having a new instance
 		// in any case, we're adding the details to the map
 		logger.Infof("GameServer %s/%s does not exist in cache, we're creating it", gameServerNamespace, gameServerName)
+		// save actual gameserver creation time
+		creationTimeStamp := obj.GetCreationTimestamp()
+
 		gsdi = &GameServerInfo{
 			GameServerNamespace: gameServerNamespace,
 			Mutex:               &sync.RWMutex{},
 			GsUid:               obj.GetUID(),
 			CreationTime:        n.nowFunc().UnixMilli(),
+			CreationTimeStamp:   &creationTimeStamp,
```

Comment:
I think this field is superfluous since we already have the `CreationTime` one. It will be *slightly* different than the CreationTimestamp (since it will take some - hopefully - milliseconds for K8s to trigger the watch and the NodeAgent to get notified) but I think it should be enough. 

CreationTime is in milliseconds, so we can easily subtract it and emit the metric. Example: https://go.dev/play/p/RAOPckRxo4V